cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/OAuthHandler.java

```diff
@@ -362,7 +365,7 @@ private static class OAuthServiceException extends Exception {
 
     void respond(HttpServiceResponder responder) {
       if (status == HttpURLConnection.HTTP_INTERNAL_ERROR) {
-        printStackTrace();
+        LOG.error("Internal error occurred", this);
```

Comment:
nit: `LOG.error("An internal error has occurred", this);`

Reply:
Done